Fabric.Identity.API/web.config

```diff
@@ -12,13 +12,40 @@
     </modules>
     <handlers>
       <remove name="WebDAV" />
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="dotnet" arguments=".\Fabric.Identity.API.dll" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout" >
+    <aspNetCore processPath="dotnet" arguments=".\Fabric.Identity.API.dll" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout" hostingModel="outofprocess">
       <environmentVariables>
         <clear />
         <environmentVariable name="HostingOptions__UseIis" value="true" />
         <environmentVariable name="IssuerUri" value="" />
+        <environmentVariable name="ASPNETCORE_HOSTINGSTARTUPASSEMBLIES" value="Microsoft.AspNetCore.ApplicationInsights.HostingStartup" />
+        <environmentVariable name="COMPLUS_ForceENC" value="1" />
+        <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Development" />
+        <environmentVariable name="UseDiscoveryService" value="true" />
+        <environmentVariable name="SigningCertificateSettings__PrimaryCertificateThumbprint" value="EBC1D4A73E8061E78F2157C3D618544A464C299E" />
+        <environmentVariable name="HostingOptions__UseTestUsers" value="false" />
+        <environmentVariable name="HostingOptions__StorageProvider" value="SqlServer" />
+        <environmentVariable name="SigningCertificateSettings__UseTemporarySigningCredential" value="false" />
+        <environmentVariable name="ConnectionStrings__IdentityDatabase" value="Server=HC2353.hqcatalyst.local;Database=Identity;Trusted_Connection=True;MultipleActiveResultSets=True;" />
+        <environmentVariable name="IdentityServerConfidentialClientSettings__Authority" value="http://localhost/IdentityDev" />
+        <environmentVariable name="SigningCertificateSettings__EncryptionCertificateThumbprint" value="EBC1D4A73E8061E78F2157C3D618544A464C299E" />
+        <environmentVariable name="DiscoveryServiceEndpoint" value="https://HC2353.hqcatalyst.local/DiscoveryService/v1" />
+        <environmentVariable name="AllowLocalLogin" value="false" />
+        <environmentVariable name="IdentityServerApiSettings__ApiSecret" value="secret" />
+        <environmentVariable name="IdentityServerConfidentialClientSettings__ClientSecret" value="secret" />
+        <environmentVariable name="AzureActiveDirectorySettings__Scope__1" value="profile" />

```

Comment:
All but the top three environmentvariables, shouldn't be checked in.

Reply:
This should be fixed in master, but I will make sure of that before merging.